launcher/ui/widgets/ModFilterWidget.cpp

```diff
@@ -148,11 +151,10 @@ ModFilterWidget::ModFilterWidget(MinecraftInstance* instance, bool extended, QWi
     connect(ui->forge, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
     connect(ui->fabric, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
     connect(ui->quilt, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
-
-    connect(ui->neoForge, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
-    connect(ui->forge, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
-    connect(ui->fabric, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
-    connect(ui->quilt, &QCheckBox::stateChanged, this, &ModFilterWidget::onLoadersFilterChanged);
+    if (provider == ModPlatform::ResourceProvider::FLAME)
+        ui->liteLoader->setEnabled(false);
```

Comment:
```suggestion
        ui->liteLoader->setVisible(false);
```
I personally I think hiding the checkbox outright is the better movve

Reply:
Sure, my original intention is to show that we don't miss the checkbox, it's just unavailable in this provider. I will change it to not show at all if that's what is preferred.